product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api/1.1.2-create-rest-api-by-import-oas-document/src/test/java/org/wso2/am/scenario/tests/rest/api/creation/RESTApiCreationUsingOASDocTestCase.java

```diff
@@ -189,6 +209,35 @@ public void createApiWithValidOAS3DocumentAsYAMLFile() throws Exception {
         HttpResponse getResponse = apiPublisher.getAPI(apiName, "admin", apiVersion);
         String resource = (new JSONObject(getResponse.getData())).getJSONObject("api").get("resources").toString();
         Assert.assertTrue(resource != "null", "API resource was not imported correctly");
+
+        //Assert resources
+        assertPOSTResource(((JSONArray)((JSONArray)(new JSONObject(getResponse.getData())).getJSONObject("api").
+                get("templates")).get(0)));
+        assertGETResource(((JSONArray)((JSONArray)(new JSONObject(getResponse.getData())).getJSONObject("api").
+                get("templates")).get(1)));
+    }
+
```

Comment:
shall we do some minor formatting changes for line 220 - 240:

Reply:
Done. Thanks